src/Aspire.Hosting.Azure/AzureDeployingContext.cs

```diff
@@ -23,7 +23,7 @@ namespace Aspire.Hosting.Azure;
 
 internal sealed class AzureDeployingContext(
     IProvisioningContextProvider provisioningContextProvider,
-    IUserSecretsManager userSecretsManager,
+    Publishing.IDeploymentStateManager deploymentStateManager,
```

Comment:
```suggestion
   IDeploymentStateManager deploymentStateManager,
```

And fix the using for this.